docs/module_docs/featdet_mass_detection/mass-detection.md

```diff
@@ -1,62 +1,47 @@
-# Mass detection
+# **Mass detection**
+The mass detection module generates a [mass list](../../terminology/general-terminology.md#mass-list) (_i.e._ list of m/z values and correspoinding signal intensities) for each scan, in each raw data file. During the mass detection, profile raw data are centroided and a noise filtering is performed based on a user-defined threshold (see [Setting the noise level](#setting-the-noise-level)).
 
-## LC-MS data
+---
+## **Parameters settings**
+:material-menu-open: Raw data methods → Mass detection → Mass detection
+![Mass detection](mass_detection_main.png)
 
-The mass detection steps perform noise filtering (by a threshold) and centroiding of profile raw
-data. The raw data format can either be centroided or in profile mode. If the data is centroided,
-the **centroid** mass detector can be used. Profile data requires a different mass detector such as
-**exact mass**.
+#### **Raw data files**
+Select the input raw data files for the mass detection. All the imported data files can be processed in bulk (_i.e._ _All raw data files_), or a subset can be selected directly from the _MS data files_ panel (_i.e._ _As selected in the main window_) or based on the filename (_i.e._ _File name pattern_) As an alternative, the files' directory can be also specified (_i.e._ _Specific raw data files_). Finally, if the _Those created by previous batch step_ option is selected, MZmine takes the output of the last processing step as input (particularly useful for [batch processing](../../workflows/batch_processing/batch-processing.md)
```

Comment:
_Those created by previous batch step_ is **only** applicable for batch mode and cannot be used for "normal" workflows.